sbt-plugin/src/main/scala/com/akkaserverless/sbt/AkkaserverlessPlugin.scala

```diff
@@ -48,6 +48,8 @@ object AkkaserverlessPlugin extends AutoPlugin {
       "These are the source files that are placed in the source tree, and after initial generation should typically be maintained by the user.\n" +
       "Files that already exist they are not re-generated.")
     val temporaryUnmanagedDirectory = settingKey[File]("Directory to generate 'unmanaged' sources into")
+    val onlyUnit = settingKey[Boolean]("Filters out integration tests. By default: true")
```

Comment:
Shouldn't it be through a "configuration" scope rather than a setting key, so we'd have `Test` which is the regular ones and `IntegrationTest` (already in sbt) which is the integration ones?

```scala
override def projectConfigurations = Seq(IntegrationTest)

override def projectSettings = 
  ... ++ inConfig(IntegrationTest)(only it tests)
  ++  inConfig(Test)(only unit tests)
```

Reply:
wouldn't that introduce all the problems that we have seen with sbt `it:test` ?

Reply:
FWIW I saved my WIP trying to put the integration tests into the `IntegrationTest` scope in https://github.com/lightbend/akkaserverless-java-sdk/compare/main...raboof:wip-integration-tests-into-IntegrationTest-configuration?expand=1 - but ran into various problems and abandoned that approach in favour of just adding them as regular tests. /cc @octonato

Reply:
Yeah, sorry, I had forgotten what I had heard previously about problems with that.

Still a bit annoying to have our own way for integration tests when sbt has some concept for it.

Reply:
> Still a bit annoying to have our own way for integration tests when sbt has some concept for it.

yeah, I agree - perhaps we can revisit it some day, but I reached my sbt threshold last time I tried ;)